index.md

```diff
@@ -48,7 +48,7 @@ operations/index
 people/index
 open-source/index
 finance/index
-communication/index
```

Comment:
Moving "communication" pages into the Partnerships area, because this is effectively marketing and community engagement.